javascript/gdx_getting_started_5.html

```diff
@@ -0,0 +1,96 @@
+<!--
+  This example uses the GDX JavaScript library to collect and manipulate data from a Vernier Go Direct® Sensor.
+  Press the 'Select Go Direct Device' button to open communication to the device in a Chrome browser.
+  Press the 'Download CSV' to save a copy of the data as a csv file.
+  
+  This program uses the following functions from the GDX Library:
+   
+  gdxDevice.selectDevice();
+  gdxDevice.close();
+  gdxDevice.start();
+  gdxDevice.getSensor(int);
+  sensor.setEnabled(bool);
+-->    
+
+<!doctype html>
+<html>
+<head>
+  <title>Example 5: Export Sensor Data as CSV</title>
+  <meta name="description" content="The gdx_getting_started_5.html example provides some ideas about what you can do with sensor data by exporting csv.">
+  <meta charset="utf-8">
+  <script src="https://unpkg.com/@vernier/godirect/dist/godirect.min.umd.js"></script>
+  
+</head>
+
+<body>
+  <button id="select_device">Select Go Direct Device</button>
+  <button onclick="download_csv()">Download CSV</button> 
+  
+  <pre id="output"></pre>
+ 
+  <script>
+  
+    const selectDeviceBtn = document.querySelector('#select_device');
+    const output = document.querySelector('#output');
+   
+    // create array to store data to be saved in the csv file
+    var data = [];
+
+    const selectDevice = async () => {     
+      try {
+        const gdxDevice = await godirect.selectDevice();     
+        
+        gdxDevice.start(1000);
+        
+        output.textContent += `\n Connected to `+gdxDevice.name;
```

Comment:
When I run this I do not see the device name or any data. The .csv file is empty except for a header, "Sensor Values:"